doc/lsst.ap.verify/limitations.rst

```diff
@@ -0,0 +1,12 @@
+.. py:currentmodule:: lsst.ap.verify
+
+.. _ap-verify-request:
+
+########################################
+Known Limitations 
+########################################
+
+mpSkyEphemerisQueryTask
+=======================
+
+The AP pipelines now run :py:class:`lsst.ap.association.MPSkyEphemerisQueryTask`, which includes a request to mpSky.  mpSky is not expected to host ephemerides for all test datasets, so it will often (or always) fail. When requests fail, mpSkyEphemerisQuery should return an empty DataFrame, and no detections will be associated.
```

Comment:
Please [write each sentence on a new line](https://developer.lsst.io/restructuredtext/style.html#text-wrapping); this makes it easier to spot future changes (as opposed to, say, changing one word and `git diff` counting it as replacing the whole paragraph).

Reply:
The last sentence is out of date now. 🙂